<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quize</class>
 <widget class="QMainWindow" name="Quize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мвале Лисон 4236</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>741</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>711</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="ans1">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ans2">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ans3">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="que1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>691</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GUAP TEST</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>731</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>711</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="ans4">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ans5">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ans6">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="que2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>691</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>740</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>731</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="ans7">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ans8">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ans9">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="que3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>691</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Points scored: </string>
    </property>
   </widget>
   <widget class="QLabel" name="points">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#points{
	
	color: rgb(170, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Scores</string>
    </property>
   </widget>
   <widget class="QLabel" name="timedisplay">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>740</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#timedisplay{
	
	color: rgb(170, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <zorder>groupBox1</zorder>
   <zorder>label</zorder>
   <zorder>groupBox2</zorder>
   <zorder>pushButton</zorder>
   <zorder>groupBox3</zorder>
   <zorder>submit</zorder>
   <zorder>points</zorder>
   <zorder>lab</zorder>
   <zorder>timedisplay</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
